ssh/tailssh/incubator.go

```diff
@@ -157,8 +156,9 @@ func beIncubator(args []string) error {
 func (srv *server) launchProcess(ctx context.Context, s ssh.Session, ci *sshConnInfo, lu *user.User) (cmd *exec.Cmd, stdin io.WriteCloser, stdout, stderr io.Reader, err error) {
 	shell := loginShell(lu.Uid)
 	var args []string
+	args = append(args, "-l")
```

Comment:
oh, is this right in the RawCommand case? I guess we don't want a login shell when we run a command externally, @maisem?

Reply:
yeah we don't. Only do this when RawCommand is empty.